SourceS/sdl2_to_1_2_backports.h

```diff
@@ -753,6 +754,11 @@ inline char *SDL_GetBasePath()
 	}
 #endif
 
+#if defined(__3DS__)
+	retval = "sdmc:/3ds/devilutionx/";

```

Comment:
`retval` should be dynamically allocated, because the caller is expected to free it.

Replace with:

```c++
retval = SDL_strdup("sdmc:/3ds/devilutionx/");
```